src/templates/state.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * This is an example of how to create a template that makes use of streams data.
+ * The stream data originates from Yext's Knowledge Graph. When a template in
+ * concert with a stream is built by the Yext Pages system, a static html page
+ * is generated for every corresponding (based on the filter) stream document.
+ *
+ * Another way to think about it is that a page will be generated using this
+ * template for every eligible entity in your Knowledge Graph.
+ */
+
+import * as React from "react";
+import {
+  GetHeadConfig,
+  GetPath,
+  GetRedirects,
+  HeadConfig,
+  Template,
+  TemplateConfig,
+  TemplateProps,
+  TemplateRenderProps,
+  TransformProps,
+} from "@yext/pages";
+import { isProduction } from "@yext/pages/util";
+import "../index.css";
+import Favicon from "../assets/images/yext-favicon.ico";
+import About from "../components/About";
```

Comment:
A lot of unused imports here...we should remove them here and the other templates

Reply:
Updated